doc/manual.rst

```diff
@@ -125,18 +124,18 @@ compiler may instead choose to allow the program to die with a fatal error.
     echo "invalid index"
 
 The current implementation allows to switch between these different behaviors
-via ``--panics:on|off``. When panics are turned on, the program dies on a
+via ``--panics:on|off``. When panics are turned on, the program dies in a
```

Comment:
Not sure about this one too. For me it seems like the meaning here is - when a panic happens, the program dies, so it's phrased as "the program dies on a panic"

Reply:
I've had another good look into this one too. From what I've seen, if you search Google, "dies on a panic" appears only in... Nim's documentation. My version - "dies in a panic" - doesn't seem very natural either. Given what I've seen in other language documentations, the most common way to say that is "dies with a panic".

So, I'm re-changing it :)